tests/test_tron_tools.py

```diff
@@ -178,15 +183,10 @@ def test_get_action_config(
         assert action_config == tron_tools.TronActionConfig(
             service=expected_service,
             instance=tron_tools.compose_instance('my_job', 'normal'),
-            config_dict={
-                'name': 'normal',
-                'command': 'echo first',
-                'cluster': expected_cluster,
```

Comment:
I think this would pass if you just took out 'cluster' from this dict.
I guess this works, but only because action_dict is mutable. It gets changed in the course of format_tron_action_dict (the service and deploy group get added in some cases) so I think it's better to assert on a separate dict.

Reply:
Got it.